src/hooks/useTitleChanges.js

```diff
@@ -6,6 +6,10 @@ function useTitleChanges({ noteId, title, setTitle, serviceClient }) {
   const onLocalTitleChange = useCallback(
     serviceClient
       ? e => {
+          // Let the user tab out of the title when using the 'Enter' key
+          if (e.nativeEvent?.inputType === 'insertLineBreak')
+            return window.document.querySelector('.ProseMirror').focus()
```

Comment:
This is not what we asked. We want a "tab", not a focus on this selector

Reply:
But there is no API to ask a tab event to the browser. The best we can do is find the next focusable element (tabindex), but some focusable elements do not have tabindex. What would you suggest?

Reply:
https://developer.mozilla.org/fr/docs/Web/API/EventTarget/dispatchEvent
```
window.dispatchEvent(new KeyboardEvent('keydown', {
  'key': 9
}));
```

will work, no?

Reply:
It does, that's smart

Reply:
@Crash-- updated